cpp/tests/reshape/interleave_columns_tests.cpp

```diff
@@ -374,4 +368,298 @@ TYPED_TEST(FixedPointTestBothReps, FixedPointInterleave)
   }
 }
 
+namespace {
+using StrListsCol = cudf::test::lists_column_wrapper<cudf::string_view>;
```

Comment:
+1. I'm glad you've switched out of using macros for this.

Reply:
I just want to enforce consistency. In my other PRs, I had to use macro for unknown types that come from `TYPED_TEST` tests. For example:
```
#define FWC cudf::test::lists_column_wrapper<TypeParam>;
```

Reply:
> I had to use macro for unknown types...

```c++
TYPED_TEST(...)
{
    using FWC = cudf::test::lists_column_wrapper<TypeParam>;
}
```
Oh, you mean that you'd have to define it in every test? I would *personally* prefer the verbosity over the use of a macro, but it would be good to consult the team for a different perspective.
On the bright side, these macros are local to your test file, and not in a common header. So the splash damage is minimal.